tests/test_create.py

```diff
@@ -592,3 +592,37 @@ def test_shortcut_absent_when_condarc_set(self):
             rmtree(prefix, ignore_errors=True)
             if isfile(shortcut_file):
                 os.remove(shortcut_file)
+
+    @pytest.mark.skipif(on_win, reason="gawk is a windows only package")
+    def test_search_gawk_not_win(self):
+        with make_temp_env("python") as prefix:
```

Comment:
I'm not sure you need `make_temp_env` here.  It's probably safer to keep it though to isolate and reset the config.  What you for sure don't need is the `"python"` part.  Just make an empty env.